nilearn/_utils/exceptions.py

```diff
@@ -66,3 +66,20 @@ def message(self):
 
     def __str__(self):
         return self.message
+
+
+class AllVolumesRemovedError(Exception):
+    """Warns the user if no volumes were kept.
+
+    Exception is raised when all volumes are scrubbed, i.e.,
+    `sample_mask` is an empty array.
+    """
+
+    def __init__(self):
+        super().__init__(
+            "All volumes were scrubbed, can not proceed. "
+            "The size of the sample mask is 0."
```

Comment:
I would prefer the wording here to be more consistent with the below run time warning, and state what is happening with the data first, and then give the implication:
```suggestion
            "The size of the sample mask is 0. "
            "All volumes were marked as motion outliers, can not proceed. "
```